docs/API.md

```diff
@@ -451,43 +466,29 @@ Example response:
 
 An object with the following properties:
 
-* `"client_id"`: A free-form string, optional.
-* `"domain"`: A *domain name*, required.
-* `"profile"`: A *profile name*, optional.
-* `"client_version"`: A free-form string, optional.
-* `"nameservers"`: A list of *name server* objects, optional.
-* `"ds_info"`: A list of *DS info* objects, optional.
+* `"client_id"`: A free-form string, optional. Used to monitor which client uses the API.
+* `"domain"`: A *domain name*, required. The zone to test.
+* `"profile"`: A *profile name*, optional. Used to perform the test with a specific set of parameters and tests.
+* `"client_version"`: A free-form string, optional. Used to monitor which client use the API
+* `"nameservers"`: A list of *name server* objects, optional. Used to perform un-delegated test.
+* `"ds_info"`: A list of *DS info* objects, optional. Used to perform un-delegated test.
 * `"advanced"`: **Deprecated**. A boolean, optional.
-* `"ipv6"`: A boolean, optional. (default `false`)
-* `"ipv4"`: A boolean, optional. (default `false`)
+* `"ipv6"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv4 tests.
+* `"ipv4"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv6 tests.
 * `"config"`: A string, optional. The name of a *config profile*.
-* `"user_ip"`: A ..., optional.
-* `"user_location_info"`: A ..., optional.
-* `"priority"`: A *priority*, optional
-* `"queue"`: A *queue*, optional
-
->
-> TODO: Clarify the data type of the following `"params"` properties:
-> `"user_ip"` and `"user_location_info"`.
->
-> TODO: Clarify the purpose of each `"params"` property.
->
-> TODO: Clarify the default value of each optional `"params"` property.
->
+* `"user_ip"`: An IP, optional. Used to monitor information about the user. (We only keep the location of the IP).
+* `"user_location_info"`: An *location* object, optional. Used to monitor information about the user. 
+* `"priority"`: A *priority*, optional.
+* `"queue"`: A *queue*, optional.
```

Comment:
We should put a limit of what character to use here. And lenght.

Reply:
@matsduf Check the *queue* data type specification.

The *queue* data type specification should be updated with a range restriction though, and possibly a clarification on what precision the number is rounded to.